api/specs.proto

```diff
@@ -264,6 +264,9 @@ message ContainerSpec {
 	// task will exit and a new task will be rescheduled elsewhere. A container
 	// is considered unhealthy after `Retries` number of consecutive failures.
 	HealthConfig healthcheck = 16;
+
+	// SecurityProfiles specifies per-architecture security options required by the container
+	SecurityProfiles security_profiles = 21;
```

Comment:
I'm not sure what the right wording should be.  My understanding is similar, in that was that these would be low level, and security profiles would be high level grants that would map to low level capabilities.  

I believe the current plan is to mapped on the CLI side, but possibly we'd need support for the mapping later in swarm (@aluzzardi brought up the point that it's easier to change types and versions of things in swarm than it is to change CLI option behavior).